### ISSUE TITLE
BSPWM Mouse Actions

### ISSUE THREAD

**Issue by DevMSri** [baskerville/bspwm#1512]
I am new to BSPWM so a good way to start is to use a pre-existing dotfile I am currently using gh0stzk dotfiles, which are amazing. But they don't allow me to drag or resize windows using my mouse, which is how I did it in Hyprland, instead, I have to use the alt key. How can this mouse action be integrated into BSPWM to allow for the same thing the alt key does?

**Comment by ortango** [comment]
i do not know about hyprland, but if this is done using the border as the handle; you can try this https://github.com/baskerville/bspwm/compare/master...ortango:bspwm:borderactions
it's a hack - but works well enough.

**Comment by DevMSri** [comment]
No, I meant if I could drag and resize windows with just my mouse, then start + mouse left is to drag/move whilst start + mouse right is to resize the window.

**Comment by archusXIV** [comment]
Why not trying [xdotool](https://github.com/jordansissel/xdotool?tab=readme-ov-file#sending-keys) to emulate a modifier key stroke + your mouse button{1,2,3} in a script that you call from your sxhkdrc?

**Comment by jvvv** [comment]
In the manual page, the parts about pointer_modifier, pointer_action<n>, and the section POINTER BINDINGS should help. If you are looking to avoid the pointer_modifier, I think @archusXIV's idea is valid.
